$NetBSD: patch-ac,v 1.2 2012/06/11 00:17:58 gdt Exp $

This patch was added in 2003, to "make this work with rpcgen that
exports _rpcsvcdirty incorrectly".

It is not known to be reported upstream.

--- rpcrig/rpcrigd.c.orig	2012-01-08 02:53:57.000000000 +0000
+++ rpcrig/rpcrigd.c
@@ -43,6 +43,12 @@
 
 #include <hamlib/rig.h>
 
+#ifdef __NetBSD__
+#include <sys/param.h>
+#if (__NetBSD_Version__ < 106210000)
+int _rpcsvcdirty;
+#endif
+#endif
 
 
 /*
